src/migrations/20181218115000_worker_task_ttl.js

```diff
@@ -11,6 +11,7 @@ import {safeDropIndex} from './migration_helpers';
 
 // eslint-disable-next-line import/prefer-default-export
 export const up = async (db, {workerTasksTTLInSeconds}, logger) => {
+  await db.collection('workerTasks').remove({startTime: {$exists: false}});
```

Comment:
```suggestion
  await db.collection('workerTasks').deleteMany({startTime: {$exists: false}});
```

Reply:
Also, this line should be a separate migration. Moving a existing migration to the end of the queue, is not a good idea. Other migrations written since, may depend on it being run before them.